src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -209,9 +209,8 @@ private boolean areAllChildrenIgnored() {
      * Returns a {@link Statement}: run all non-overridden {@code @BeforeClass} methods on this class
      * and superclasses before executing {@code statement}; if any throws an
      * Exception, stop execution and pass the exception on.
-     * @param testClass 
      */
-    public static Statement withBeforeClasses(Statement statement, TestClass testClass) {
+    protected Statement withBeforeClasses(Statement statement) {
```

Comment:
Curious why you want to call these directly vs. calling `classBlock()`.  If you cannot use `classBlock()` then perhaps instead of making these three public we should extract one protected method that calls all three so 1) your runner isn't hard-coding the order and 2) we can add more class (ex `withClassFixtures()`). Perhaps call the new method `withClassStatements()`